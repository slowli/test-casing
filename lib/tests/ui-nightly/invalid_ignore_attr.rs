use test_casing::test_casing;

#[test_casing(2, ["test", "this"])]
#[ignore(_arg > 1)]
fn tested_function(_arg: &str) {
    // Does nothing
}

#[test_casing(2, ["test", "this"])]
#[ignore(message = "???")]
fn other_tested_function(_arg: &str) {
    // Does nothing
}

fn main() {}
