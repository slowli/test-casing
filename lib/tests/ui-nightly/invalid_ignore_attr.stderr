error: unrecognized attribute shape; should have `#[ignore] or `#[ignore = "value"]` form
 --> tests/ui-nightly/invalid_ignore_attr.rs:4:1
  |
4 | #[ignore(_arg > 1)]
  | ^^^^^^^^^^^^^^^^^^^

error: unrecognized attribute shape; should have `#[ignore] or `#[ignore = "value"]` form
  --> tests/ui-nightly/invalid_ignore_attr.rs:10:1
   |
10 | #[ignore(message = "???")]
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^
